src/dispatch.jl

```diff
@@ -150,3 +150,75 @@ function plot_electric_dispatch(d::Dict; title="Electric Systems Dispatch", save
     PlotlyJS.plot(traces, layout)  # will not produce plot in a loop
 
 end
+
+function plot_thermal_dispatch(d::Dict; title="Thermal Systems Dispatch", save_html=false)
+
+    traces = PlotlyJS.GenericTrace[]
+
+    layout = PlotlyJS.Layout(
+        title_text = "",
+        xaxis_title_text = "time step",
+        yaxis_title_text = "kW",
+        yaxis2 = (
+            title = "temp [C]",
+            overlaying = "y",
+            side = "right"
+        )
+    )
+
+    if "ExistingChiller" in keys(d)
+        T = length(d["ExistingChiller"]["year_one_electric_consumption_series"])
+    else
+        T = 8760
```

Comment:
perhaps we should make `T` a function argument too? There are so many different cases for which thermal technologies could be in the results dictionary that it would be onerous to handle all of them in if/else statements.

Reply:
Makes sense, will make it an input arg and default to 8760